vuepress/docs/next/tutorials/devops/backing-restoring-keycloak.md

```diff
@@ -0,0 +1,50 @@
+---
+sidebarDepth: 2
+---
+
+# Backing up and Restoring Keycloak
+
+
+This tutorial describes how to backup and restore an active Entando Identity Management Keycloak instance when you need to migrate or restore the realm to another environment.
+
+## Prerequisites:
+A Keycloak instance running on Entando with a database management system (DBMS) in the backend. This procedure will not work with an embedded database. 
+
+## Create a Backup of Keycloak Realm
+1. From the command line, get the pod name of the Keycloak server you need to backup.
+2. Shell into your VM and execute this command to connect to the pod.
+```
+ kubectl exec -it your-keycloak-pod-name -- /bin/bash
+ ```
+3. Create a new directory in /tmp with:
+```
+mkdir -p /tmp/export 
+ ```
+
+4. Run the following script to retrieve all the data and save it to a JSON file in the export directory. A new Keycloak server will run on a different port (offset=200) to avoid conflicts with the original Entando Keycloak instance.
+```
+/opt/jboss/keycloak/bin/standalone.sh
+-Djboss.socket.binding.port-offset=200
+-Dkeycloak.migration.action=export 
+-Dkeycloak.migration.provider=singleFile 
+-Dkeycloak.migration.realmName=entando
+-Dkeycloak.migration.usersExportStrategy=DIFFERENT_FILES 
+-Dkeycloak.migration.file=/tmp/export/entando-prod-realm.json
+```
+5. If there are no errors, press Ctrl+C to stop the process.
+6. Type `exit` to go back to the VM layer.
+7. Execute this command to copy the realm data file to the proper location.  
+```
+kubectl cp [keycloak-pod-name]:/tmp/export/ keycloak-backup
+```
+## Import the Keycloak Realm
+1. Login to the original Entando Keycloak server for which you just created the backup.
+2. From the left sidebar, go to Import.
+ 
+3. Click `Select file` button and `upload` the JSON file from the keycloak-backup directory. You will see the following page:
+
+![Entando ID Management UI](./img/import-keycloak.png)
+ 
+::: Note
+If we are using a different environment, with a different hostname, we should import only users or set the If for resource exists to Skip
```

Comment:
@nshaw Not sure what to make of this.

Reply:
"if we are" -> "if you are" & "we should" -> "you should"

"only users" -> "users only"

format "If for resource exists" (pretty sure that tag/option is labeled slightly differently ??)

Reply:
"set the flag for 'If a resource exists` to `Skip`"

Reply:
it should technically be placed after; i looked it up :P i can point you the references and we can discuss in the style meeting?

Reply:
I think as an adverb, it is technically ok here? But we can discuss :)

Reply:
no worries, this is grammatically correct! into the weeds but something i was curious about